### PR TITLE
Arhiver for Haiku 

### PR DESCRIPTION
**Pull request by punsith**

### CODE REVIEW

**Comment by scottmc** [comment]
There is an option in Paladin to export a Makefile.  Do that and then the compiling step might be as easy as  running "make".

**Comment by punsith** [comment]
okay :)

**Comment by punsith** [comment]
i updated :)

**Comment by richienyhus** [comment]
@scottmc & @waddlesplash any further changes needed ?

File: Source/Archiver.h [hunk 1]
```diff
@@ -51,7 +51,7 @@ SOFTWARE
 #include <View.h>
 #include <Window.h>
 
-#include <be/add-ons/tracker/TrackerAddon.h>
+#include </boot/system/develop/headers/os/add-ons/tracker/TrackerAddOn.h>
```
Comment (pulkomandy) [inline Source/Archiver.h:54]: This is not needed, keeping just <add-ons/tracker/TrackerAddOn.h> should work.

File: Source/Archiver.h [hunk 2]
```diff
@@ -63,7 +63,7 @@ SOFTWARE
 
 #define	ARCHIVER_SETTINGS_FILE_PATH		"/boot/home/config/settings/"
 #define	ARCHIVER_SETTINGS_FILE			"archiver.settings"
-#define	ARCHIVER_RULES_FILE_PATH		"/boot/home/config/etc/"
+#define	ARCHIVER_RULES_FILE_PATH		"/boot/home/config/non-packaged/etc/"
```
Comment (pulkomandy) [inline Source/Archiver.h:66]: I think it is better to look in the packaged directory? Or does it try to write here?
Ideally we should look in both but it is out of the scope of your task

File: drag _Archiver-A_ here [hunk 3]
```diff
@@ -1 +1 @@
-/boot/home/config/add-ons/Tracker
\ No newline at end of file
+/boot/system/settings/etc
```
Comment (pulkomandy) [inline drag _Archiver-A_ here]: Are you sure this is where it goes? If it is a tracker add-on, then the proper place would be /boot/home/config/non-packaged/add-ons/Tracker

**Review by pulkomandy — CHANGES_REQUESTED** [review]

File: Source/Archiver.h (hunk 1 above)
Comment (punsith) [reply Source/Archiver.h:54]: mm but it didn't compiled 
"boot/home/Archiver/Source/Archiver.h:54: /add-ons/tracker/TrackerAddOn.h: No such file or directory
/boot/system/develop/etc/makefile-engine:291: recipe for target 'objects.x86-gcc2-release/Archiver.o' failed
make: *** [objects.x86-gcc2-release/Archiver.o] Error 1
"

File: Source/Archiver.h (hunk 2 above)
Comment (punsith) [reply Source/Archiver.h:66]: user should paste the rules file to that location but the program don't use the file on that location as Pete Goodeve said ,  it using rules file in " B_SYSTEM_ETC_DIRECTORY" . and also user should paste rules file on their but if I comment that  path it won't compile :/ but when i delete the rules file from non-packaged nothing changed :/  it works fine

**Comment by scottmc** [comment]
Any progress on cleaning this up?

**Comment by punsith** [comment]
@scottmc 
 "ARCHIVER_RULES_FILE_PATH		"/boot/home/config/non-packaged/etc/""  
is something I cant understand. The archiver don't us the location above. It using .rules file in B_SYSTEM_ETC_DIRECTORY. but if I dete the  that code it wont work.

**Pull request closed without merging**